test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -41,44 +38,58 @@
+            "Tested bug occurs only on-paper printing " +
+                    "so you mustn't use PDF printer\n\n" +
+                    "1. Java print dialog should appear.\n" +
                     "2. Press the Print button on the Java Print dialog.\n" +
-                    "3. Please check the page margin rectangle are properly drawn and visible on all sides on all pages.\n" +
+                    "3. Please check the page margin rectangle are properly " +
+                    "drawn and visible on all sides on all pages.\n" +
                     "If so, press PASS, else press FAIL.\n\n" +
-                    "Also you may run this test in paper-saving mode. Due to tested bug affects pages printed with transparency in LANDSCAPE and REVERSE_LANDSCAPE orientations " +
-                    "there is an option to print only 2 pages affected. To do it pass PaperSavingMode parameter.";
+                    "Also you may run this test in paper-saving mode. Due " +
+                    "to tested bug affects pages printed with transparency " +
+                    "in LANDSCAPE and REVERSE_LANDSCAPE orientations there " +
+                    "is an option to print only 2 pages affected. " +
+                    "To do it pass PaperSavingMode parameter.";
 
     private static boolean isPaperSavingMode = false;
 
     public static void main(String[] args) throws Exception {
         if (PrinterJob.lookupPrintServices().length > 0) {
 
-            String instructionsHeader = "This test prints 6 pages with page margin rectangle and a text message. \n";
+            String instructionsHeader = "This test prints 6 pages with page " +
+                    "margin rectangle and a text message. \n";
             if (args.length > 0) {
                 isPaperSavingMode = args[0].equals("PaperSavingMode");
             }
             if (isPaperSavingMode) {
-                instructionsHeader = "This test prints 2 pages with page margin rectangle and a text message. \n";
+                instructionsHeader = "This test prints 2 pages with page " +
+                        "margin rectangle and a text message. \n";
             }
-            PassFailJFrame.builder().rows(15).instructions(instructionsHeader + INSTRUCTIONS)
-                    .testUI(() -> createTestUI()).build().awaitAndCheck();
+            PassFailJFrame.builder().instructions(instructionsHeader + INSTRUCTIONS)
+                    .rows(15).testUI(() -> createTestUI()).build().awaitAndCheck();
 
         } else {
-            System.out.println("Test failed : Printer not configured or available.");
-            throw new RuntimeException("Test failed : Printer not configured or available.");
+            System.out.println("Test failed : " +
+                    "Printer not configured or available.");
+            throw new RuntimeException("Test failed : " +
+                    "Printer not configured or available.");
```

Comment:
I'd rather keep those unmodified, the string is not so long, and wrapping it doesn't improve readability.

In my opinion, `println` is redundant, the runtime exception conveys it.